src/views/Profile/PersonalInfo/PersonalInfo.tsx

```diff
@@ -249,189 +315,272 @@ export const PersonalInfo: React.FC<IEditAuthorProps> = ({
+                    onChange={(event) =>
+                      inputSocialNetworkChangeHandler(event, index)
+                    }
+                    onBlur={blurHandler}
+                  />
+                </Box>
+              );
+            })}
+          </Grid>
+          <Grid item xs={8}>
+            <Box className={classes.WrapperBox}>
+              <InputLabel required className={classes.InputLabel}>
+                {i18n.t(langTokens.admin.mainPlaceOfWork)}
+              </InputLabel>
+              {visitFields.mainWorkingPlace &&
+                errorMessages.mainWorkingPlace && (
+                  <Typography className={classes.Typography}>
+                    {errorMessages.mainWorkingPlace}
+                  </Typography>
+                )}
+            </Box>
+            <TextField
+              variant="outlined"
+              required
+              fullWidth
+              name="mainWorkingPlace"
+              value={newAuthorValues.mainWorkingPlace}
+              onChange={inputChangeHandler}
+              onBlur={blurHandler}
+            />
+            <Box className={classes.WrapperBox}>
+              <InputLabel required className={classes.InputLabel}>
+                {i18n.t(langTokens.admin.biography)}
+              </InputLabel>
+              {visitFields.bio && errorMessages.bio && (
+                <Typography className={classes.Typography}>
+                  {errorMessages.bio}
+                </Typography>
+              )}
+            </Box>
+            <TextField
+              multiline
+              minRows={12}
+              variant="outlined"
+              required
+              fullWidth
+              name="bio"
+              value={newAuthorValues.bio}
+              onChange={inputChangeHandler}
+              onBlur={blurHandler}
+            />
+          </Grid>
         </Grid>
-      </Grid>
 
-      <Box className={classes.ButtonBox}>
-        {author && !isCurrentUser && (
-          <CancelButton
-            label={i18n.t(langTokens.common.cancelChanges)}
-            onClick={() => window.close()}
+        <Box className={classes.ButtonBox}>
+          {author && !isCurrentUser && (
+            <CancelButton
+              label={i18n.t(langTokens.common.cancelChanges)}
+              onClick={() => window.close()}
+            />
+          )}
+          <BasicButton
+            disabled={isSaveDisabled}
+            type="sign"
+            label={i18n.t(langTokens.common.acceptChanges)}
+            className={classes.BasicButton}
+            onClick={buttonClickHandler}
           />
-        )}
-        <BasicButton
-          disabled={isSaveDisabled}
-          type="sign"
-          label={i18n.t(langTokens.common.acceptChanges)}
-          className={classes.BasicButton}
-          onClick={buttonClickHandler}
-        />
-      </Box>
-    </form>
+        </Box>
+      </form>
+      {isAdmin && author && !isCurrentUser && (
```

Comment:
is it easy enough to move this part into separate component?

Reply:
I will try